reactor-netty-http/src/main/java/reactor/netty/http/server/HttpServer.java

```diff
@@ -686,11 +686,39 @@ public final HttpServer route(Consumer<? super HttpServerRoutes> routesBuilder)
 	 * @return a new {@link HttpServer}
 	 */
 	public final HttpServer secure(Consumer<? super SslProvider.SslContextSpec> sslProviderBuilder) {
+		return secure(sslProviderBuilder, false);
+	}
+
+	/**
+	 * Apply an SSL configuration customization via the passed builder. The builder
+	 * will produce the {@link SslContext} to be passed to with a default value of
+	 * {@code 10} seconds handshake timeout unless the environment property {@code
+	 * reactor.netty.tcp.sslHandshakeTimeout} is set.
+	 * <p>
+	 * If {@link SelfSignedCertificate} needs to be used, the sample below can be
+	 * used. Note that {@link SelfSignedCertificate} should not be used in production.
+	 * <pre>
+	 * {@code
+	 *     SelfSignedCertificate cert = new SelfSignedCertificate();
+	 *     SslContextBuilder sslContextBuilder =
+	 *             SslContextBuilder.forServer(cert.certificate(), cert.privateKey());
+	 *     secure(sslContextSpec -> sslContextSpec.sslContext(sslContextBuilder), true);
+	 * }
+	 * </pre>
+	 *
+	 * @param sslProviderBuilder  builder callback for further customization of SslContext.
+	 * @param redirectHttpToHttps true enables redirecting HTTP to HTTPS on the same port.
+	 *                            This configuration is applicable only for HTTP/1.x.
```

Comment:
```suggestion
	 * @param redirectHttpToHttps true enables redirecting HTTP to HTTPS by changing the
	 *                            scheme only but otherwise leaving the port the same.
	 *                            This configuration is applicable only for HTTP 1.x.
```

Reply:
updated.